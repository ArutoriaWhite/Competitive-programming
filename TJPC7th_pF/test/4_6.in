18296682 978
17006 1
24582 0
47252 1
59418 1
70637 0
113091 1
141783 1
177271 1
183100 0
188048 1
190651 1
196657 0
228962 0
231016 0
237458 0
240224 0
260911 1
262126 1
262543 0
302019 1
304467 1
361145 0
364526 0
369821 0
395700 1
423499 0
442358 0
443616 0
453750 0
473436 1
511440 0
540015 1
555703 0
558767 0
597413 1
608618 0
615614 1
629574 0
661095 0
676427 1
678674 0
714527 1
747132 0
750031 0
753591 1
773329 1
798798 0
811280 0
813891 1
814791 1
850937 1
895809 0
898094 1
934380 0
964543 1
978677 0
983906 0
1000133 0
1005608 1
1015864 1
1017894 0
1028076 0
1028527 1
1079972 0
1134834 1
1139632 0
1159124 0
1166278 0
1182131 0
1192393 1
1207603 1
1258462 0
1261200 0
1291374 0
1305512 1
1314179 0
1353680 0
1354720 1
1355508 0
1394948 0
1406200 1
1410064 1
1420200 1
1423038 0
1427884 1
1450247 1
1471091 1
1506125 1
1509196 0
1512551 1
1585653 0
1585673 0
1611999 1
1634301 1
1663905 0
1672632 0
1695097 1
1706080 1
1715176 0
1730607 0
1833843 0
1835845 1
1852367 1
1852800 0
1884847 1
1914370 1
1931588 0
1937465 1
1948947 1
1960942 1
1988741 1
2012960 0
2013754 1
2014033 1
2033591 1
2067620 0
2094009 1
2105374 1
2120420 0
2148588 0
2149791 1
2155289 1
2179006 0
2182085 1
2278182 1
2316391 1
2341558 1
2373482 0
2374581 0
2396491 1
2449615 1
2456275 0
2466421 1
2494075 1
2517729 1
2522501 1
2527248 1
2527461 0
2535657 0
2542700 1
2598242 0
2599789 0
2622924 0
2657645 0
2676577 1
2748582 0
2773937 1
2828492 1
2855541 0
2871946 1
2895533 0
2897589 0
2898846 1
2901188 0
2911587 1
2947733 0
2989076 1
2991734 1
3013691 0
3019174 1
3052672 0
3073809 0
3099989 0
3133596 0
3137455 1
3184213 0
3193146 1
3203252 1
3208850 0
3225114 0
3250213 0
3251900 0
3264585 1
3267014 0
3270048 1
3270565 1
3284994 0
3290231 1
3298493 1
3301046 0
3344333 1
3346046 0
3366605 0
3370271 1
3392786 0
3396850 0
3427232 0
3433137 0
3463210 1
3486572 1
3498063 1
3499257 1
3521944 1
3526899 1
3563129 0
3568458 0
3570913 1
3571254 0
3625762 1
3641698 0
3641955 1
3650131 1
3659942 1
3662707 1
3663244 0
3694269 1
3696007 1
3718624 0
3723029 0
3758023 0
3795311 0
3808570 0
3944879 1
3956074 1
3972233 1
3999704 0
4001105 0
4003448 1
4004675 1
4006851 1
4013971 1
4033874 1
4037029 0
4085613 1
4097261 1
4100531 0
4128736 0
4140457 0
4157787 1
4185701 0
4203428 1
4217998 0
4259213 1
4263983 1
4309114 1
4355553 1
4360359 1
4377738 1
4378392 1
4380670 0
4397674 1
4401067 0
4427528 1
4441135 1
4443273 1
4446620 0
4534596 1
4544694 1
4545262 0
4545642 1
4551535 0
4552944 1
4563386 0
4568140 1
4612097 1
4623577 1
4639603 1
4688119 0
4693514 1
4706740 1
4724892 1
4734097 0
4743297 0
4775203 0
4789349 1
4858515 0
4864474 1
4892959 1
4894709 1
4900523 0
4900904 0
4903932 0
4910647 0
4935409 1
4942262 1
4972996 0
4999449 1
5041707 1
5045994 0
5078850 0
5117783 0
5129828 1
5137466 1
5138482 0
5149172 0
5150184 1
5161843 1
5198255 1
5200038 0
5209622 1
5249319 1
5261559 1
5308420 0
5319323 1
5342552 1
5365341 1
5392315 0
5394462 0
5408439 0
5431866 0
5455686 1
5534282 0
5570519 0
5571505 0
5594408 0
5603607 0
5633275 1
5670439 1
5675690 1
5677178 1
5699592 0
5756183 0
5759381 1
5795767 0
5796924 0
5806774 1
5810737 0
5835348 0
5905815 1
5928328 0
5984820 1
5998083 1
6015907 1
6084211 1
6084767 1
6099945 1
6121540 0
6125448 1
6128311 1
6182481 1
6184462 0
6192270 0
6311848 1
6317099 0
6336937 1
6345427 1
6400392 0
6417709 1
6440402 1
6443131 1
6452527 1
6462870 0
6467545 0
6471833 1
6492615 0
6493450 0
6505636 1
6516034 0
6516698 1
6557617 1
6591454 0
6607046 1
6616671 0
6622509 0
6643629 1
6645020 1
6645884 1
6653923 1
6667378 1
6698193 0
6751232 1
6753229 0
6755172 0
6769508 0
6811518 0
6815724 0
6821131 1
6828453 0
6847277 1
6872728 0
6876532 1
6887964 1
6900879 0
6906209 0
6930336 1
6941712 0
6953736 1
6961941 1
6980899 0
7028085 1
7028562 0
7040618 1
7050146 1
7052549 0
7150082 1
7156864 1
7165954 0
7172521 0
7174169 1
7209665 1
7264359 1
7307065 1
7330747 0
7342144 1
7342278 0
7375483 1
7386638 1
7470618 1
7499920 1
7514034 0
7547487 0
7567546 1
7635338 1
7638807 1
7642342 0
7671735 1
7680707 1
7691206 0
7708114 1
7711098 0
7711672 1
7718403 1
7719705 1
7737706 1
7740513 0
7753013 1
7770892 0
7785111 1
7786930 1
7820521 0
7824952 1
7881198 1
7888062 1
7895230 0
7897029 0
7932104 1
7959395 0
7968673 0
7970377 1
7982888 0
7988134 1
8001032 1
8006247 0
8033859 1
8037793 1
8041170 0
8053165 0
8054827 1
8071787 0
8085706 1
8097205 0
8101536 0
8102944 0
8103467 0
8105670 1
8126024 1
8162705 1
8173301 0
8175608 1
8186961 1
8189238 0
8195750 0
8198413 1
8228067 0
8228853 1
8229727 1
8265512 1
8282185 0
8288551 0
8327946 1
8330210 0
8333755 1
8336288 1
8362668 1
8396689 0
8397890 0
8482708 0
8484345 1
8499960 0
8557790 0
8691308 1
8729456 0
8730333 1
8752264 1
8762937 1
8826429 0
8848598 0
8848863 1
8865143 1
8882434 1
8888152 0
8892572 0
8907697 0
8955425 1
8973631 1
8975124 0
8993480 0
9002508 0
9006041 0
9016960 1
9043757 0
9056460 1
9077633 0
9129681 0
9191164 0
9228841 0
9260366 1
9280589 1
9315865 1
9393429 1
9398381 1
9408125 1
9417911 1
9430215 1
9436386 0
9453494 1
9471699 1
9473099 0
9482035 0
9486626 1
9498443 1
9510646 1
9540231 0
9556694 1
9580410 1
9589168 0
9618167 0
9661763 0
9669152 0
9669359 0
9676729 1
9687942 1
9690791 1
9740183 0
9758017 0
9762846 1
9778331 0
9779993 1
9792951 0
9819727 0
9826566 1
9826643 0
9829989 1
9868560 1
9876623 0
9938985 0
9947359 0
9967596 0
9992744 0
10016984 0
10026213 1
10033857 0
10056667 1
10082593 1
10102736 1
10119255 0
10123093 1
10158680 1
10163179 1
10169660 0
10182832 1
10184042 1
10216184 0
10219864 0
10231310 1
10240132 1
10241395 0
10248586 0
10254882 0
10263805 1
10271138 0
10273497 0
10284026 1
10291823 0
10294809 1
10295988 0
10306934 0
10309105 0
10319742 0
10353075 0
10363354 1
10414519 0
10415818 1
10456495 0
10459077 0
10460910 1
10499401 1
10512813 1
10572094 1
10579197 1
10591107 1
10600640 1
10641798 1
10697565 0
10700764 0
10721232 1
10758820 1
10777309 0
10807833 0
10823898 0
10836436 0
10852680 1
10858108 0
10880328 0
10892529 0
10917145 1
10945552 1
10956335 1
10961545 0
10998057 0
11057374 0
11068253 0
11090135 0
11137242 1
11150322 0
11172352 0
11178585 1
11223037 1
11230471 0
11238435 0
11288526 0
11318832 1
11336510 1
11360369 1
11378708 0
11388001 0
11449158 1
11469686 1
11470893 0
11566641 0
11582802 1
11584663 1
11586502 0
11598261 0
11598940 1
11600351 0
11611680 1
11681833 0
11697409 0
11712988 0
11724409 1
11734223 0
11793500 0
11797825 0
11816032 1
11823852 1
11825317 1
11842036 0
11854054 1
11874619 1
11898731 0
11985180 0
11988488 0
12003659 0
12024425 0
12031718 0
12035901 0
12085684 1
12117119 1
12126443 1
12143445 0
12173835 0
12175384 0
12185290 0
12241665 1
12249167 1
12254606 0
12276968 1
12330600 0
12344776 0
12349152 0
12351963 1
12366613 1
12378592 0
12379419 0
12381748 1
12393832 0
12434284 0
12438545 1
12445890 1
12496804 0
12508980 1
12534744 1
12564293 1
12574341 0
12577574 0
12598893 0
12612965 0
12620478 1
12647263 1
12650121 1
12659562 0
12691532 0
12705507 0
12717944 0
12718907 0
12776950 0
12806505 1
12807327 1
12811140 1
12816155 1
12821596 0
12836600 0
12899357 1
12906474 1
12915153 1
12941201 1
12945935 0
12967868 0
12968499 0
12985887 1
12987166 1
13059215 0
13064126 1
13077060 0
13094423 0
13125837 0
13131883 1
13135797 0
13144176 0
13156344 0
13183662 0
13228847 1
13251895 0
13394139 0
13394992 1
13406175 1
13462721 0
13469377 0
13511634 1
13519805 1
13538178 1
13548241 1
13549712 1
13594439 1
13601153 0
13628556 1
13630763 1
13642100 0
13655354 1
13660358 0
13696594 1
13698633 1
13738730 0
13741698 1
13769359 0
13775615 0
13777091 1
13798094 1
13807613 1
13812380 1
13848914 1
13860052 1
13863170 1
13886227 1
13895573 0
13898772 0
13920639 1
13922025 1
14001938 0
14009899 1
14062354 1
14074463 1
14125333 1
14126416 1
14201694 0
14238321 0
14239510 1
14246298 0
14264039 0
14273210 1
14279188 1
14288870 1
14302050 0
14310431 0
14396900 1
14452706 0
14454945 0
14457552 1
14459206 1
14467906 1
14470588 1
14509615 1
14528644 1
14559091 0
14607557 1
14614223 0
14615597 1
14652847 1
14654428 1
14703903 1
14714665 1
14717041 1
14763537 1
14784307 1
14785198 0
14787268 0
14792786 1
14796694 0
14812367 0
14861418 1
14889519 1
14922518 0
14932183 0
14952725 0
14957605 0
14961719 0
15031668 1
15040621 1
15053208 0
15059005 1
15062430 0
15076284 0
15096621 1
15108763 0
15109894 0
15110939 0
15112932 1
15114772 0
15125208 0
15126665 1
15128107 1
15131227 0
15142189 0
15148083 1
15149990 1
15153575 0
15162980 1
15170197 1
15188678 1
15212193 1
15249744 0
15265416 1
15287656 0
15321630 1
15326137 1
15385210 1
15421851 0
15454221 1
15470045 1
15485431 1
15498755 1
15518675 1
15541254 0
15584832 0
15593560 1
15611976 1
15634907 1
15637124 0
15640102 0
15642782 0
15651104 0
15654146 1
15673850 1
15684018 1
15738199 1
15759419 0
15786722 1
15787241 0
15797518 1
15814316 1
15837052 0
15840829 0
15869881 0
15904931 0
15922201 1
15926963 1
15938054 0
15944619 0
15945290 1
15983259 1
15986528 1
15995418 1
16019081 1
16025269 0
16032512 1
16054531 1
16084203 1
16109369 1
16118994 1
16139353 0
16143669 1
16158250 1
16173877 1
16176331 0
16219458 1
16265187 1
16281381 0
16315114 1
16345434 1
16357995 0
16365652 1
16391010 1
16391447 1
16432908 1
16440290 1
16468798 0
16476116 1
16477167 0
16478041 0
16480752 1
16487626 1
16502760 0
16516156 0
16529093 0
16605672 1
16637618 0
16643370 1
16682826 0
16688619 0
16690373 0
16692475 1
16721670 1
16787682 1
16803256 1
16817873 0
16829496 1
16848997 1
16914001 1
16934356 0
16951515 0
16974338 0
16983273 1
16986783 1
17007443 0
17028140 1
17031074 1
17033300 1
17065739 0
17070328 0
17081640 1
17083145 0
17090977 0
17146337 1
17147195 1
17149963 0
17150234 1
17159270 1
17162560 1
17172514 1
17209490 1
17308653 0
17315299 0
17342365 0
17371999 1
17395377 1
17408988 0
17428866 1
17453767 1
17462308 1
17480529 0
17509353 0
17549581 0
17553281 0
17574206 1
17619557 0
17628302 0
17638238 1
17647493 0
17653044 0
17681769 1
17704799 0
17719430 1
17743173 0
17813921 1
17813937 0
17824541 1
17834345 1
17850420 1
17884942 1
17890160 1
17896750 1
17899642 1
17913399 1
17949055 1
17958180 0
17971701 1
17972506 1
17973070 0
17991207 1
17999909 1
18019803 1
18040364 1
18044149 1
18047528 1
18113523 0
18138698 0
18285613 1
