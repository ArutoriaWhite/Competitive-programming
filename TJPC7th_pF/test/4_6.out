9260366 18120662
